docs/get-started/Validator-Node.md

```diff
@@ -1,6 +1,6 @@
 # Running a Validator Node
 
-Before setting up your validator node, make sure you've already installed  **Iris** by this [guide](Full-Node.md)
+Before setting up your validator node, make sure you've already installed  `Iris` by this [guide](Full-Node.md) and your node is fully synced.
 
```

Comment:
already installed `iris` by **following** this guide...

Reply:
fixed